nestkernel/nest.h

```diff
@@ -82,6 +82,9 @@ ArrayDatum get_connections( const DictionaryDatum& dict );
 
 void simulate( const double& t );
 void resume_simulation();
+void run( const double& t );
+void prepare();
+void cleanup();
```

Comment:
Pointer to an example of the doxygen documentation? It hasn't been used in this header previously, so I'm not precisely sure without an example.